tests/testData.ts

```diff
@@ -1,4 +1,6 @@
 export enum Wallets {
   richWalletAddress = "0x36615Cf349d7F6344891B1e7CA7C72883F5dc049",
   richWalletPrivateKey = "0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110",
```

Comment:
You will need to perform some refactoring if you merge this. But please make the naming consistent for the wallets here: 

```suggestion
  firstWalletAddress = "0x36615Cf349d7F6344891B1e7CA7C72883F5dc049",
  firstWalletPrivateKey = "0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110",
```